core/src/main/java/gov/nist/secauto/metaschema/core/model/constraint/AbstractTargetedConstraints.java

```diff
@@ -40,7 +38,7 @@
  *          the Java type of the constraint container
  */
 public abstract class AbstractTargetedConstraints<T extends IValueConstrained>
```

Comment:
Non-blocking: I will run through the code and try to make sure I understand better while I do a more thorough review, but is there such a thing with external constraints an "untargeted" constraint. Reading the Javadocs in another class:

```java
 * Represents a set of constraints that target a given definition using a target
 * Metapath expression.
```

Is there an alternative or opposite to this situation such we need to name this? I think I am missing something obvious but wanted to check.

Reply:
All constraints are targeted at a definition. Do you have a suggestion for a better name?

Reply:
Let me mull this over too tonight, but probably not. I guess it is "a given" but maybe that is a good reason to keep it in the name. Maybe there is some tactical addition to class documentation to help that, and I will look at the Javadocs and think about if adding something to this base class will help me and those like me in the future. If not, I will resolve and close.

Reply:
Ditto here, I guess I am good for now.